etc/check_env.sh

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env bash
-
-if [[ ! $TOPOLOGY =~ ^(server|replica_set|sharded_cluster)$ ]]; then
-    >&2 echo "Invalid value of TOPOLOGY. TOPOLOGY must be set to one of: server, replica_set, sharded_cluster"
-    exit 1
-fi
-
-if [ ! -z $AUTH ] && [[ ! $AUTH =~ ^(noauth|auth)$ ]]; then
```

Comment:
We don't need any of the other checks?

Reply:
Whoops - didn't mean to remove the whole file. Added it back without the `TOPOLOGY` check in the new commit.